jsonnet/dashboards/grafana-templates.libsonnet

```diff
@@ -206,6 +218,19 @@ local template = grafana.template;
         label='Logs Search',
       ),
 
+    retentionTemplate()::
+      template.text(
+        name='retention',
+        label='Retention',
+      ) {
+        current: {
+          selected: false,
+          text: '999',
```

Comment:
I send you example with `999` just for illustration. You should set in to some reasonable value.